### PR TITLE
[Capacity update] US + minor change to EIA capacity parser

### PR DESCRIPTION
**Pull request by mathilde-daugy**
## Issue
New capacity data needed to filter out flaky EIA parser data 

## Description
Update to EIA parser so that we have capacity for all production modes
Update US yaml files with capacity data from 2023-10-01

### Double check

- [ ] I have tested my parser changes locally with `poetry run test_parser "zone_key"`
- [ ] I have run `pnpx prettier --write .` and `poetry run format` to format my changes.

### CODE REVIEW

File: config/zones/US-FLA-HST.yaml
Comment (mathilde-daugy) [inline config/zones/US-FLA-HST.yaml:1]: Manually reallocated oil as gas because there is apparently only gas production from EIA parser

**Review by VIKTORVAV99 — APPROVED** [review]

**Comment by mathilde-daugy** [comment]
@VIKTORVAV99 i'm reviewing all the values now, will merge after. :)

**Comment by VIKTORVAV99** [comment]
> @VIKTORVAV99 i'm reviewing all the values now, will merge after. :)

Don't forget to run prettier, otherwise it won't let you merge.

Easiest way would be `pnpx prettier@2 --write .` in the top level repo.

File: electricitymap/contrib/config/model.py
[the remark this answers is not in the document]
Comment (VIKTORVAV99) [reply electricitymap/contrib/config/model.py:1]: It should probably be a float anyway!

**Review by VIKTORVAV99 — APPROVED** [review]

**Pull request merged**